webapp/src/components/kanban/kanban.tsx

```diff
@@ -46,6 +50,20 @@ const vStrength = createVerticalStrength(Utils.isMobile() ? 60 : 250)
 
 const Kanban = (props: Props) => {
     const {board, activeView, cards, groupByProperty, visibleGroups, hiddenGroups} = props
+    const limitedCard = useAppSelector(getHiddenCard)
+    const intl = useIntl()
+    const hiddenCardGroupId = 'hidden-card-group-id'
+
+    if (limitedCard.length > 0) {
+        props.hiddenGroups.push({
```

Comment:
I wouldn't confuse "LimitedCards" with "HiddenGroups". I would keep those things separate. If you create a "LimitedCards" component, you can simply add that compoent after the "hiddenGroups". see below, Line 331. None of this code would be necessary.